src/modules/Elsa.EntityFrameworkCore/Modules/Management/WorkflowInstanceStore.cs

```diff
@@ -198,12 +202,18 @@ private async ValueTask OnLoadAsync(ManagementElsaDbContext managementElsaDbCont
         var compressionAlgorithm = (string?)managementElsaDbContext.Entry(entity).Property("DataCompressionAlgorithm").CurrentValue ?? nameof(None);
         var compressionStrategy = _compressionCodecResolver.Resolve(compressionAlgorithm);
 
-        if (!string.IsNullOrWhiteSpace(json))
+        try
         {
-            json = await compressionStrategy.DecompressAsync(json, cancellationToken);
-            data = _workflowStateSerializer.Deserialize(json);
+            if (!string.IsNullOrWhiteSpace(json))
+            {
+                json = await compressionStrategy.DecompressAsync(json, cancellationToken);
+                data = _workflowStateSerializer.Deserialize(json);
```

Comment:
This is actually the only really necessary fix to make it so that the Cleanup script can delete WorkflowInstance(s) that have bad data in them.  Even the try/catch below in the CleanupJob only prevents the CleanupJob from crashing the entire Elsa server, but it won't allow it to cleanup bad instances without this fix.

Reply:
True, so maybe let's add an exception handler here. If an error occurred here, it wouldn't make the workflow instance useful anyway, so we might just as well handle the exception for now, swallow it, and log an error. 
Then the CleanupJob and other bulk operations can still continue normally.